rest_framework/views.py

```diff
@@ -90,10 +90,10 @@ def exception_handler(exc, context):
         if getattr(exc, 'wait', None):
             headers['Retry-After'] = '%d' % exc.wait
 
-        if isinstance(exc.detail, (list, dict)):
+        if isinstance(exc.detail, (list, tuple, dict)):
```

Comment:
Off the cuff this might make sense, but I don't think any code specifically handles tuple-based `detail`s. I would revert this because it's not related to the issue and it's not tested.

Reply:
as you know, tuple is better than list if data is immutable.
if you create APIExcpetion or Permission with iterable message, default message is immuntable so that Tubles have an advantage.

but as you said, this is not related to the issue. so i will remove this code and add new PR